sorc/build_all.sh

```diff
@@ -9,14 +9,25 @@ set +x
 #                   Anything other than "true"  will use libraries locally.
 #------------------------------------
 
-while getopts "c" option;
+_build_ufs_options=""
+
+while getopts "ac" option;
 do
  case $option in
+  a)
```

Comment:
Does the readme need to be updated for this?

Reply:
Yes. I'll get on that.

Reply:
Updated